chef/cookbooks/provisioner/templates/default/tftp.erb

```diff
@@ -8,8 +8,8 @@ service tftp
        socket_type             = dgram
        protocol                = udp
        wait                    = yes
-       flags                   = IPv6 IPv4
+       flags                   = IPv4
```

Comment:
looks like the bug is fixed already: https://build.opensuse.org/request/show/332002